EIPS/eip-3374.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 3374
+title: Predictable Proof-of-Work (POW) Sunsetting 
+author: @Query0x
+discussions-to: <tbd>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-13
+---
+
+## Simple Summary
+Provides for a predictable, issuance neutral reduction of block rewards to 1 over a one year time span while reducing the uncertainties and risks associated with sudden changes to mining economics.
+
+## Abstract
+Sets the block reward to 3 ETH and then incrementally decreases it every block for 2,362,000 blocks (approximately 1 year) until it reaches 1 ETH.
+
+## Motivation
+Unnecessarily abrupt changes to the Ethereum ecosystem cause disruption and disharmony resulting in the disenfranchisement of community members while undermining stability and confidence.  While moves from Proof-of-Work to Proof-of-Stake will undoubtedly cause friction between those community members vested in either, all benefit from a measured, predictable transition.
+
+This proposal:
```

Comment:
EIPs do not propose, they assert.  Write the EIP targeting a reader in 5 years long after this EIP is implemented.

Reply:
This EIP had better be irrelevant five years from now...